### ISSUE TITLE
[Bug]: music stops at random moments

### ISSUE THREAD

**Issue by WataLaker37821** [spartacus04/jext-reborn#100]
### Describe the bug

Hi again, can you fix a bug when music stops at random moment?

### Steps to reproduce

I just enabled jukebox gui, put disks into jukebox and start random disk, and playing can stop at a random moment.

### Expected behaviour

Music isnt supposed to stop while playing (limits with seconds were set up by me for 1000) 

### Server Version

1.18.2

### Server Type

Spigot

### Discs.json

```json
[
    {
        "title": "Odyssey",
        "author": "RazMusic",
        // duration of the disc file in seconds
        "duration": 100,
        // disc name in sounds.json, example: 
        "disc-namespace": "kevinmacleod.odyssey",
        // model data of the disc
        "model-data": 88,
        // true if allows creeper drops, false otherwise
        "creeper-drop": true,
        "lores": [
            "lore added by disc",
            "each entry is a new line"
        ],
        // loottables in which the disc can be found
        "loot-tables": [
            "chests/ancient_city"
        ],
        // loottables in which the disc fragments can be found
        "fragment-loot-tables": [
            "chests/ancient_city"
        ]
    },
    {
        "title": "Cipher",
        "author": "RazMusic",
        "duration": 100,
        "disc-namespace": "kevinmacleod.cipher",
        "model-data": 89,
        "creeper-drop": true,
        "lores": [],
        "loot-tables": [],
        "fragment-loot-tables": []
    },
    {
        "title": "Infiniteamethyst",
        "author": "RazMusic",
        "duration": 100,
        "disc-namespace": "lenaraine.infiniteamethyst",
        "model-data": 90,
        "creeper-drop": true,
        "lores": [],
        "loot-tables": [],
        "fragment-loot-tables": []
    },
    {
        "title": "AlreadyDead",
        "author": "RazMusic",
        "duration": 1000,
        "disc-namespace": "razfill.alreadydead",
        "model-data": 91,
        "creeper-drop": true,
        "lores": [],
        "loot-tables": [],
        "fragment-loot-tables": []
    }
]
```

### Resource pack

https://www.dropbox.com/s/lv0ghi7xtvbumqz/mc.RazFill.zip?dl=0

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.
- [X] I agree to the fact that if this issue becomes inactive it will be closed.

**Comment by WataLaker37821** [comment]
not stale but fresh

**Comment by spartacus04** [comment]
sorry about the bot, I just re-enabled the stale action and it marked some issues as stale, I got back to work on the plugin yesterday and I'll fix this issue too

**Comment by WataLaker37821** [comment]
> sorry about the bot, I just re-enabled the stale action and it marked some issues as stale, I got back to work on the plugin yesterday and I'll fix this issue too

no problem my friend, thanks for answer, i hope you are doing good

**Comment by spartacus04** [comment]
Hi, i just fixed this in [ee9f92f](https://github.com/spartacus04/jext-reborn/commit/ee9f92fadf1e82c261e7e1cabb7e0b45570421b4), it should be implemented in the next update
